business/definitionService.js

```diff
@@ -80,7 +81,7 @@ class DefinitionService {
   async list(coordinates, recompute = false) {
     if (!recompute) return this.definitionStore.list(coordinates)
     const curated = await this.curationService.list(coordinates)
-    const tools = await this.harvestService.list(coordinates)
+    const tools = await this.harvestStore.list(coordinates)
```

Comment:
Why the switch from `harvestService` to `harvestStore` for these?

Reply:
this was using the harvest store all along, it just had a funny name

[HarvestStore](https://github.com/clearlydefined/service/blob/master/providers/stores/azblobHarvestStore.js)
[HarvestService](https://github.com/clearlydefined/service/blob/master/providers/harvest/crawler.js)